scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -982,38 +961,39 @@ public void jobTaskStarted(final InternalJob job, final InternalTask task,
             public Void executeWork(Session session) {
                 long jobId = jobId(job);
 
-                String jobUpdate = "update JobData set status = :status, "
-                        + "startTime = :startTime, numberOfPendingTasks = :numberOfPendingTasks, "
-                        + "numberOfRunningTasks = :numberOfRunningTasks where id = :jobId";
+                String jobUpdate = "update JobData set status = :status, " +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "update JobData set status = :status, " 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)